package.json

```diff
@@ -6,11 +6,16 @@
     "type": "git",
     "url": "git://github.com/mozilla/normandy.git"
   },
+  "bin": {
```

Comment:
Figured it out - I was convinced I needed this because I couldn't run `stylelint` from the command line without it. npm runs it just fine though, so I don't even need it.